test/unit/sequence_base_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative '../test_helper'
+
+class DelayedResourceTestSequence < MiniTest::Test
+  def setup
+    allergy_intolerance_bundle = FHIR.from_contents(load_fixture(:us_core_r4_allergy_intolerance))
+    @allergy_intolerance_resource = allergy_intolerance_bundle.entry.first.resource
+    @instance = Inferno::Models::TestingInstance.new(
+      url: 'http://www.example.com',
+      client_name: 'Inferno',
+      base_url: 'http://localhost:4567',
+      client_endpoint_key: Inferno::SecureRandomBase62.generate(32),
+      client_id: SecureRandom.uuid,
+      selected_module: 'us_core_r4',
+      oauth_authorize_endpoint: 'http://oauth_reg.example.com/authorize',
+      oauth_token_endpoint: 'http://oauth_reg.example.com/token',
+      scopes: 'launch openid patient/*.* profile',
+      token: 99_897_979
+    )
+
+    @instance.save!
+
+    client = FHIR::Client.new(@instance.url)
+    client.use_r4
+    client.default_json
+    @sequence = Inferno::Sequence::USCoreR4AllergyintoleranceSequence.new(@instance, client, true)
```

Comment:
I know it is convenient to reference IG-specific sequences in our tests, but at some point we need to split this apart so that the platform doesn't depend on the existence of sequences.  Not a problem to fix now but it would be good for everyone to start thinking about Inferno the platform vs. IG-specific tests.

Reply:
That's true. Luckily it still worked when I just changed the AllergyIntolerance sequence to SequenceBase.